pyproject.toml

```diff
@@ -11,13 +11,16 @@ documentation = "https://docs.beamerbridge.com/"
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.11"
-web3 = "^5.24.0"
+web3 = "^6.0.0b7"
 click = "^8.0.3"
 structlog = "^22.3.0"
 python-statemachine = "^0.9.0"
 prometheus-client = "^0.16.0"
 lru-dict = "^1.1.7"
 toml = "^0.10.2"
+eth-ape = {git = "https://github.com/ApeWorX/ape", rev = "main"}
+ape-solidity = {git = "https://github.com/ApeWorX/ape-solidity.git", rev = "main"}
+ape-ganache = {git = "https://github.com/ApeWorX/ape-ganache.git", rev = "main"}
```

Comment:
This is fine for now, but before merging this PR, we need to make sure there's an ape release with all relevant fixes for us so that we can switch to it and not use whatever is on their `main` branch.

Reply:
pinned ape-solidity as it wont change and for others i am following

Reply:
pinned them all